content/docs/command-reference/remove.md

```diff
@@ -101,7 +101,38 @@ deleted (just the `model` file in this example):
 ```dvc
 $ dvc remove train --outs
 $ ls
-dvc.lock  dvc.yaml  foo.csv  foo.csv.dvc  train.py
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  train.py
 ```
 
 > Notice that the dependencies (`data.csv` and `train.py`) are not deleted.
+
+## Example: remove a single output file by name
+
+Imagine the same <abbr>workspace</abbr> as before but now we have multiple
+`outs` for the `train` stage:
+
+```yaml
+train:
+  cmd: python train.py data.csv
+  deps:
+    - data.csv
+    - train.py
+  outs:
+    - logs
+    - model.h5
+```
+
+```dvc
+$ ls
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  logs  model.h5  train.py
+```
+
+Using `dvc remove` you can remove a specific file using it's output name:
```

Comment:
what is the semantics of this operation? it is not clear from this update, unfortunately - do we remove outputs from the workspace? from the cache? do we update the `dvc.yaml` and/or remove .dvc along the way?

Reply:
I have tried to update the semantics for better explaining the new behavior.